src/main/java/mezz/jei/api/IModRegistry.java

```diff
@@ -92,6 +92,19 @@
 	void addDescription(List<ItemStack> itemStacks, String... descriptionKeys);
 
 	/**
+	 * Adds an anvil recipe for the given inputs and output.
+	 * @param leftInput The itemStack placed on the left slot.
+	 * @param rightInput The itemStack placed on the right slot.
+	 * @param output The resulting itemStack.
+	 * @param levelsCost The cost, in experience levels, that will be required in order to pick up the item.
+	 *                   Optional: If not specified, the cost text will not display on the recipe.
+	 */
+	void addAnvilRecipe(ItemStack leftInput, ItemStack rightInput, ItemStack output, int levelsCost);
+	void addAnvilRecipe(ItemStack leftInput, ItemStack rightInput, ItemStack output);
+	void addAnvilRecipe(ItemStack leftInput, List<ItemStack> rightInput, List<ItemStack> output, int levelsCost);
+	void addAnvilRecipe(ItemStack leftInput, List<ItemStack> rightInput, List<ItemStack> output);

```

Comment:
Is it possible to automatically determine the cost, maybe using a fake `ContainerRepair`?